db/migrate/20191012123313_add_sendable_to_message_logs.rb

```diff
@@ -0,0 +1,6 @@
+class AddSendableToMessageLogs < ActiveRecord::Migration[5.2]
+  def change
+    remove_column :message_logs, :sent_to_id, :integer
```

Comment:
I think here we need to use remove_reference (or, keep remove_column and add remove_foreign_key) so we drop the foreign_key to send_to